Anvil/Tools/Get.pm

```diff
@@ -1615,6 +1617,21 @@ sub host_name
 		}
 	}
 	
+	# If the host name is 'localhost', appeand the short UUID 
```

Comment:
`appeand`->`append`?

Reply:
I'll fix that in my branch, thanks for catching and approving.